pandas/tslib.pyx

```diff
@@ -2722,6 +2722,13 @@ class Timedelta(_Timedelta):
     __pos__ = _op_unary_method(lambda x: x, '__pos__')
     __abs__ = _op_unary_method(lambda x: abs(x), '__abs__')
 
+
+# Resolution is in nanoseconds
+# (2**63)/(1*24*60*60*(10**9)) = 106751.991167
+Timedelta.min = Timedelta(-106751.991167, 'D')
```

Comment:
Actually, looks like you need `min + 1` since `min` corresponds to `NaT`:
```
In [6]: Timedelta(np.iinfo(np.int64).min, 'ns')
Out[6]: NaT

In [7]: Timedelta(np.iinfo(np.int64).min + 1, 'ns')
Out[7]: Timedelta('-106752 days +00:12:43.145224')

In [8]: Timedelta(np.iinfo(np.int64).max, 'ns')
Out[8]: Timedelta('106751 days 23:47:16.854775')
```